cacheops/invalidation.py

```diff
@@ -168,8 +168,9 @@ def invalidate_model(model):
     if conjs_keys:
         cache_keys = redis_client.sunion(conjs_keys)
         redis_client.delete(*(list(cache_keys) + conjs_keys))
-        for k in conjs_keys:
-            redis_client.srem(set_key, k)
+
+        for batch in batcher(conjs_keys, 200):
+            redis_client.srem(set_key, *batch)
```

Comment:
Nice work with the itertools functions.

    ...could be relatively slow on large datasets

To really improve the Python performance, we may want to use `re.finditer` rather than splitting the string into a list.  Not critical now, but could be a nice optimization in the future.  This would mean *not* creating a list in memory and instead work 100% with generators.  Since your code is already doing that, I think it'd speed things up slightly.  Again, not a big deal for small data sets but when there are 100,000 objects, it could make a slight difference: https://docs.python.org/2/library/re.html#re.finditer